src/libstore/globals.hh

```diff
@@ -924,7 +924,22 @@ public:
         )"};
 
     Setting<std::set<ExperimentalFeature>> experimentalFeatures{this, {}, "experimental-features",
-        "Experimental Nix features to enable."};
+        R"(
+          Experimental Nix features to enable. Current experimental features
+          are the following:
+
+          - `ca-derivations`
```

Comment:
It would be better to generate this list dynamically from `stringifiedXpFeatures` in `experimental-features.cc`. (That map could even be extended with some Markdown documentation for each feature.)

Reply:
Thank you for the feedback! I've added some changes that dynamically populate the contents of the docs, along with a quick summary of each experimental feature. Some of them seem pretty obscure, so please correct me if I got any of them wrong.